src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/SubmitEventForCaseWorkerConsumerTest.java

```diff
@@ -1,113 +0,0 @@
-package uk.gov.hmcts.reform.wataskmonitor.consumer.ccd;
-
-import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
-import au.com.dius.pact.consumer.junit5.PactTestFor;
-import au.com.dius.pact.core.model.RequestResponsePact;
-import au.com.dius.pact.core.model.annotations.Pact;
-import org.json.JSONException;
-import org.junit.jupiter.api.Test;
-import org.springframework.http.HttpHeaders;
-import org.springframework.http.MediaType;
-import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
-import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
-import uk.gov.hmcts.reform.wataskmonitor.consumer.ccd.util.CcdConsumerTestBase;
-
-import java.util.Map;
-
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.Assert.assertNotNull;
-import static org.springframework.http.HttpHeaders.AUTHORIZATION;
-import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
-import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
-import static uk.gov.hmcts.reform.wataskmonitor.consumer.ccd.util.PactDslBuilderForCaseDetailsList.buildSubmitForCaseWorkedPactDsl;
-
-public class SubmitEventForCaseWorkerConsumerTest extends CcdConsumerTestBase {
-
-    @Override
-    public Map<String, Object> setUpStateMapForProviderWithCaseData(CaseDataContent caseDataContent)
-        throws JSONException {
-        Map<String, Object> caseDataContentMap = super.setUpStateMapForProviderWithCaseData(caseDataContent);
-        caseDataContentMap.put(EVENT_ID, SUBMIT_APPEAL);
-        return caseDataContentMap;
-    }
-
-    @Pact(provider = "ccdDataStoreAPI_Cases", consumer = "wa_task_monitor")
-    public RequestResponsePact submitEventForCaseWorker(PactDslWithProvider builder) throws JSONException {
-        return builder
-            .given("A Submit Event for a Caseworker is requested",
-                setUpStateMapForProviderWithCaseData(caseDataContent)
-            )
-            .uponReceiving("A Submit Event for a Caseworker")
-            .path(buildPath())
-            .query("ignore-warning=true")
-            .method("POST")
-            .matchHeader(AUTHORIZATION, AUTH_TOKEN)
-            .matchHeader(SERVICE_AUTHORIZATION, SERVICE_AUTH_TOKEN)
-            .matchHeader(CONTENT_TYPE, APPLICATION_JSON_VALUE)
-            .body(convertObjectToJsonString(createCaseDataContent(SUBMIT_APPEAL, caseDetailsMap)))
-            .matchHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
-            .willRespondWith()
-            .status(201)
-            .body(buildSubmitForCaseWorkedPactDsl(CASE_ID))
-            .toPact();
-    }
-
-    @Test
-    @PactTestFor(pactMethod = "submitEventForCaseWorker")
-    public void verifySubmitEventForCaseworker() throws Exception {
-
-        final CaseDetails caseDetails = coreCaseDataApi.submitEventForCaseWorker(
```

Comment:
This is used currently in GivensBuilder

        coreCaseDataApi.submitEventForCaseWorker(
            userToken,
            serviceToken,
            userInfo.getUid(),
            "IA",
            "Asylum",
            caseDetails.getId().toString(),
            true,
            submitCaseDataContent
        );

This pact test needs to be reintroduced. This is used in functional test.

Reply:
okay got it, i can retain them